ext/standard/exec.c

```diff
@@ -272,8 +269,7 @@ PHP_FUNCTION(passthru)
    Escape all chars that could possibly be used to
    break out of a shell command
 
-   This function emalloc's a string and returns the pointer.
-   Remember to efree it when done with it.
+   This function allocates a new zend_string, remember to free it when done.
```

Comment:
well technically alloc can be also persistent, emalloc and efree are not (unless zend alloc disabled) so it might be a bit less clear now. Not a big deal as we usually don't have any comments.